installer/bundle/bundle_skel.sh

```diff
@@ -784,15 +767,19 @@ case "$installMode" in
         if [ $scx_installed -ne 0 -a $omi_installed -ne 0 ]; then
             echo "Installing OMS agent ..."
 
-            pkg_add_list $OMI_PKG omi
-            pkg_add_list $SCX_PKG scx
             pkg_add_list $OMS_PKG omsagent
 
             python_ctypes_installed
             if [ $? -eq 0 ]; then
                 pkg_add_list $DSC_PKG omsconfig
             fi
 
+            # Install SCX (and OMI)
+            [ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+            [ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
+            ./bundles/${SCX_INSTALLER} --install $FORCE $DEBUG
+            SCX_STATUS=$?
+
             # Now actually install of the "queued" packages
             if [ -n "${add_list}" ]; then
                 if [ "$INSTALLER" = "DPKG" ]; then
```

Comment:
When I was trying to repro the errors (upgrade using omsagent1.2.0-148 on ubuntu 14) I noticed that omsconfig was trying to start omsagent before any package were configured. It didn't caused any failures but I thought this is something we should avoid and do a package wise update. Not sure it will improve anything (also I haven't tried it with latest omsagent bundles) but just wanted to bring out my observation.